packages/twenty-front/src/generated-metadata/graphql.ts

```diff
@@ -468,54 +468,39 @@ export type EnvironmentVariable = {
 };
 
 export enum EnvironmentVariablesGroup {
-  Authentication = 'Authentication',
```

Comment:
not related to this PR but could we take a look at the CI, this should be detected